kbase-extension/static/kbase/js/kbaseNarrative.js

```diff
@@ -174,63 +172,83 @@ define([
+            token: this.getAuthToken(),
+        });
+        return ws.get_workspace_info({ id: this.workspaceId })
             .then((wsInfo) => {
                 return wsInfo[5];
             });
-    }
-
-    /**
-     * A wrapper around the Jupyter.notebook.kernel.execute() function.
-     * If any KBase widget needs to make a kernel call, it should go through here.
-     * ...when it's done.
-     */
-    Narrative.prototype.executeKernelCall = function () {
-        console.info('no-op for now');
     };
 
     // Wrappers for the Jupyter/Jupyter function so we only maintain it in one place.
     Narrative.prototype.patchKeyboardMapping = function () {
-        var commonShortcuts = [
-                'a', 'm', 'f', 'y', 'r',
-                '1', '2', '3', '4', '5', '6',
-                'k', 'j', 'b', 'x', 'c', 'v',
-                'z', 'd,d', 's', 'l', 'o', 'h',
-                'i,i', '0,0', 'q', 'shift-j', 'shift-k',
-                'shift-m', 'shift-o', 'shift-v'
+        const commonShortcuts = [
+                'a',
+                'm',
+                'f',
+                'y',
+                'r',
+                '1',
+                '2',
+                '3',
+                '4',
+                '5',
+                '6',
+                'k',
+                'j',
+                'b',
+                'x',
+                'c',
+                'v',
+                'z',
+                'd,d',
+                's',
+                'l',
+                'o',
+                'h',
+                'i,i',
+                '0,0',
+                'q',
+                'shift-j',
+                'shift-k',
+                'shift-m',
+                'shift-o',
+                'shift-v',
             ],
             commandShortcuts = [],
             editShortcuts = [
                 // remove the command palette
-                // since it exposes commands we have "disabled"
+                // since it exposes commands we have 'disabled'
                 // by removing keyboard mappings
                 'cmdtrl-shift-p',
             ];
 
         commonShortcuts.forEach(function (shortcut) {
             try {
                 Jupyter.keyboard_manager.command_shortcuts.remove_shortcut(shortcut);
-            } catch (ex) {
-                console.warn('Error removing shortcut "' + shortcut + '"', ex);
+            } catch (e) {
+                console.warn('Error removing shortcut "' + shortcut + '"', e);
             }
             try {
-                Jupyter.notebook.keyboard_manager.edit_shortcuts.remove_shortcut(shortcut);
-            } catch (ex) {
-                // console.warn('Error removing shortcut "'  + shortcut +'"', ex);
+                Jupyter.notebook.keyboard_manager.edit_shortcuts.remove_shortcut(
+                    shortcut
+                );
+            } catch (e) {
+                // console.warn('Error removing shortcut "' + shortcut + '"', e);
```

Comment:
maybe not relevant to this PR but should this still be commented out or do we want to do anything with the error?

Reply:
That big honkin' file needs a redo in a later PR. I think I mentioned that in the notes in a different one. Let's get this merged, and I'll work on that.